app/views/pages/community_moderation.html.erb

```diff
@@ -49,8 +49,8 @@
     <h3>Using our quick emoji rating to down-rank content</h3>
     <p>Clicking on the 'thumbsdown' or 'vomit' button will activate them. Here's what each emoji does: </p>
     <ol>
-      <li>Thumbsdown(👎): a quick way to “push content down" so that it’s less likely to be seen on the site (i.e. articles that only has a teaser paragraph and sends the reader to an external link for the rest). </li>
-      <li>Vomit(🤢):a quick way to flag content that violates the <a href="https://dev.to/code-of-conduct">Code of Conduct</a> or <a href="https://dev.to/terms">Terms</a> so the DEV core team are notified and can taken action.</li>
+      <li>Thumbsdown(👎): a quick way to “push content down" so that it’s less likely to be seen on the site (i.e. articles that only have a teaser paragraph and sends the reader to an external link for the rest). </li>
+      <li>Vomit(🤢):a quick way to flag content that violates the <a href="https://dev.to/code-of-conduct">Code of Conduct</a> or <a href="https://dev.to/terms">Terms</a> so the DEV core team are notified and can take action.</li>
```

Comment:
A super small thing, but would you mind adding a space between the colon `:` and `a` after the vomit emoji?

Reply:
Heh. Looks like https://github.com/thepracticaldev/dev.to/pull/2393/files got it.

Reply:
Thanks I just saw that it has been fixed.